create3_control/controllers/fb_linearization.py

```diff
@@ -1,16 +1,32 @@
+# Copyright 2022 Ilario Antonio Azzollini.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 import numpy as np
 
 from geometry_msgs.msg import Vector3
 from geometry_msgs.msg import Twist
 
 from create3_control.controllers.controller_interface import ControllerInterface
-import create3_control.utilities as utils
+from create3_control.utilities import add_relative_to_absolute_pose as add_rel_to_abs_pose
```

Comment:
do not `as` functions/classes names.
this makes it harder to search them in the repository

Reply:
ok